apps/condo/domains/ticket/schema/Incident.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Generated by `createschema ticket.Incident 'organization; number; details:Text; status; textForResident:Text; workStart:DateTimeUtc; workFinish:DateTimeUtc; isScheduled:Checkbox; isEmergency:Checkbox; hasAllProperties:Checkbox;'`
+ */
+
+const dayjs = require('dayjs')
+
+const { GQLError } = require('@open-condo/keystone/errors')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { storeChangesIfUpdated, buildSetOfFieldsToTrackFrom } = require('@condo/domains/common/utils/serverSchema/changeTrackable')
+const { normalizeText } = require('@condo/domains/common/utils/text')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+const access = require('@condo/domains/ticket/access/Incident')
+
+const { OMIT_INCIDENT_CHANGE_TRACKABLE_FIELDS } = require('../constants')
+const { INCIDENT_ERRORS } = require('../constants/errors')
+const { INCIDENT_STATUSES, INCIDENT_STATUS_ACTUAL } = require('../constants/incident')
+const { createIncidentChange, incidentRelatedManyToManyResolvers, incidentChangeDisplayNameResolversForSingleRelations } = require('../utils/serverSchema/IncidentChange')
+
+
+const Incident = new GQLListSchema('Incident', {
+    schemaDoc: 'Entries of scheduled and emergency incidents of water, electricity, etc.',
+    fields: {
+
+        organization: ORGANIZATION_OWNED_FIELD,
+
+        number: {
+            schemaDoc: 'Autogenerated incident human readable ID',
+            type: 'AutoIncrementInteger',
+            kmigratorOptions: { unique: true, null: false },
+        },
+
+        details: {
+            schemaDoc: 'Text description of the incident',
+            type: 'Text',
+            isRequired: true,
+            kmigratorOptions: { null: false },
+            hooks: {
+                resolveInput: async ({ resolvedData, fieldPath }) => {
+                    return normalizeText(resolvedData[fieldPath])
+                },
+            },
+        },
+
+        status: {
+            schemaDoc: 'Incident status.',
+            type: 'Select',
+            isRequired: true,
+            options: INCIDENT_STATUSES.join(','),
+            defaultValue: INCIDENT_STATUS_ACTUAL,
+        },
+
+        textForResident: {
+            schemaDoc: 'Text that employees should say to residents',
+            type: 'Text',
+            hooks: {
+                resolveInput: async ({ resolvedData, fieldPath }) => {
+                    return normalizeText(resolvedData[fieldPath])
+                },
+            },
+        },
+
+        workStart: {
+            schemaDoc: 'Date and time of works start',
```

Comment:
Is it the same as incident start time? or not?

Reply:
Yes, the same. Probably should be renamed to `startDate` and `endDate`

Reply:
Updated docs